scm/src/main/java/com/sandboni/core/scm/GitRepository.java

```diff
@@ -22,24 +22,32 @@
 
 public class GitRepository implements GitInterface {
     private final RevisionResolver revisionResolver;
-    private final ChangeScopeResolver changeScopeResolver;
     private final BlameResolver blameResolver;
+    private final Repository repository;
```

Comment:
This seems to be an old version of the class. Did you merge the latest develop branch into yours?

Reply:
Bad pull, will update PR now. Thanks